src/ui/Controls/SETextBox.cs

```diff
@@ -344,7 +348,7 @@ public int SelectionLength
                 }
                 else if (_uiTextBox != null)
                 {
-                    var target = _uiTextBox.SelectionLength;
+                    var target = value;
```

Comment:
This is for the "Selection Lenght" issue, I don't know it it was intended the way it was.

Reply:
That's a plain bug :)